_includes/github_issue.md

```diff
@@ -0,0 +1 @@
+[{{ include.repo }}#{{ include.number }}](https://github.com/{{ include.repo }}/issues/{{ include.number }}){:target="_blank" rel="noopener" class="_"}
```

Comment:
FWIW; I know we're using it in many places, but the "class" option may not be needed; seems just to make sure the link doesn't have a CSS class assigned (we should even consider doing the reverse, and use a special class for these links)

Reply:
Yes many places: https://github.com/docker/docker.github.io/search?q=rel%3D%22noopener%22+class%3D%22_%22 :sweat_smile:. Let's track this for a follow-up.